server/types/resolvable.go

```diff
@@ -31,152 +31,89 @@ import (
 // It is used for domain types, and it can be used
 // for other user-defined types we don't support yet.
 type ResolvableType struct {
-	Typ tree.ResolvableTypeReference
+	Typ          tree.ResolvableTypeReference
```

Comment:
I don't recall seeing this type used anywhere, but I may have missed it. Are we still using this?